assets/js/applepay.js

```diff
@@ -0,0 +1,122 @@
+jQuery( document ).ready(
+	function($) {
+		$( 'form.checkout' ).on(
+			'checkout_place_order',
+			function() {
+				const payment_method = $( 'form.checkout input[name="payment_method"]:checked' ).val();
+
+				if ($.inArray( payment_method, applepay_ajax_obj.apple_pay_terminal ) > -1) {
+					onApplePayButtonClicked( payment_method );
+					return false;
+				}
+			}
+		);
+	}
+);
+
+function onApplePayButtonClicked(payment_method) {
+
+	if ( ! ApplePaySession) {
+		return;
+	}
+
+	// Define ApplePayPaymentRequest
+	const request = {
+		"countryCode": applepay_ajax_obj.country,
+		"currencyCode": applepay_ajax_obj.currency,
+		"merchantCapabilities": [
+			"supports3DS"
+		],
+		"supportedNetworks": [
+			"visa",
+			"masterCard",
+			"amex",
```

Comment:
You might want to have this as configuration in the plugin (in the UI), but with default `visa, masterCard, amex`

Reply:
Didn't we do that for Magento and NetSuite?

Reply:
> You might want to have this as configuration in the plugin (in the UI), but with default `visa, masterCard, amex`

Yes, I have also created a configuration field for these on the plugin side and added selected`visa, masterCard, amex` options by default 
![Screenshot 2023-03-17 at 3 37 29 PM](https://user-images.githubusercontent.com/39646587/225881852-5e7be1dc-e3c6-45be-9e76-b4273031e81d.png)

Reply:
> Didn't we do that for Magento and NetSuite?

No, we hardcoded the values there as well. But I will create tickets for each to get them dynamically.